### DISCUSSION TITLE
PC311C-W-TY Used to work but no longer

### DISCUSSION THREAD

**Discussion by botas11** [make-all/tuya-local#1895] [Q&A] [answered]
After HA core update the PC311C-W-TY that I had connected started showing "Unavailable", tried readding it but no luck even if I have the local key and everything updated. It only shows the following:

![image](https://github.com/make-all/tuya-local/assets/52585024/943fefb4-2edc-42e5-99cc-98d231bb6f0e)

Does anyone knows how to solve this problem?

**Comment by make-all** [comment]
Did you check for tuya-local updates?

**Reply by botas11** [reply]
yes, I have the latest version but still no luck
![image](https://github.com/make-all/tuya-local/assets/52585024/99b37624-2bae-44f3-b038-a83da175c094)

**Reply by botas11** [reply, marked as answer]
Ok, seems the problem was that the auto protocol was not working had to choose it manually

**Comment by make-all** [comment]
This seems like a connection issue, so #522 is worth looking at, maybe the firmware got updated and you need to use a different protocol version, or double check the IP address and local key.
